domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraUseCase.kt

```diff
@@ -51,11 +53,15 @@ interface CameraUseCase {
     suspend fun takePicture()
 
     /**
-     * Takes a picture with the camera. If the imageCaptureUri is not null, save the picture at
-     * the location of the url on the device. If the imageCaptureUri is null, save the picture in
-     * the picture directory.
+     * Takes a picture with the camera. If ignoreUri is set to true, the picture taken will be saved
+     * at the default directory for pictures on device. Otherwise, it will be saved at the uri
+     * location if the uri is not null. If it is null, an error will be thrown.
      */
-    suspend fun takePicture(contentResolver: ContentResolver, imageCaptureUri: Uri?)
+    suspend fun takePicture(
+        contentResolver: ContentResolver,
+        imageCaptureUri: Uri?,
+        ignoreUri: Boolean = false
+    ): CompletableDeferred<ImageCapture.OutputFileResults>
```

Comment:
We shouldn't be returning `CompletableDeferred` from a `suspend` function. If anything it should be `ImageCapture.OutputFileResults`.